assets/components/button/button.jsx

```diff
@@ -4,17 +4,17 @@
 
 import React from 'react';
 
-import DangerouslySetButtonOnAnyElement, { defaultProps, type PropTypes } from './dangerouslySetButtonOnAnyElement';
+import DangerouslySetButtonOnAnyElement, { defaultProps, type GenericPropTypes } from './dangerouslySetButtonOnAnyElement';
 import './button.scss';
 
 // ----- Render ----- //
 
-type AllPropTypes = {
-  ...PropTypes,
+type PropTypes = {
+  ...GenericPropTypes,
```

Comment:
The name `GenericPropTypes` loses its meaning when imported into a different file. I would suggest <name of component>GenericPropTypes, or rename it when you import it.